assemblies/assembly-enabling-authentication.adoc

```diff
@@ -1,38 +1,45 @@
 [id='enabling-authentication']
 = Enabling authentication in {product}
 
-Authentication within {product} facilitates user sign-in, identification, and access to external resources.
-It supports multiple authentication providers.
+Depending on your organization's security policies, you might require to identify and authorize users before giving them access to resources, such as {product}.
 
-Authentication providers are typically used in the following ways:
+In {product-short}, authentication and authorization are two separate processes:
 
-- One provider for sign-in and identification.
-- Additional providers for accessing external resources.
+. Authentication defines the user identity, and passes on this information to {product-short}.
+Read following chapters to configure authentication in {product-short}.
 
-The {product} supports the following authentication providers:
+. Authorization defines what the authenticated identity can access or do in {product-short}.
+See link:{authorization-book-url}[{authorization-book-title}].
 
-//Auth0:: `auth0`
-//Atlassian:: `atlassian`
-Microsoft Azure::
-`microsoft`
-//Microsoft Azure Easy Auth:: `azure-easyauth`
-//Bitbucket:: `bitbucket`
-//Bitbucket Server:: `bitbucketServer`
-//Cloudflare Access:: `cfaccess`
-GitHub:: `github`
-//GitLab:: `gitlab`
-//Google:: `google`
-//Google IAP:: `gcp-iap`
-Keycloak:: `oidc`
-//Okta:: `okta`
-//OAuth 2 Custom Proxy:: `oauth2Proxy`
-//OneLogin:: `onelogin`
-//SAML:: `saml`
+To explore {product-short} features, you can enable the guest user to skip configuring authentication and authorization, log in as the guest user, and access all the features.
```

Comment:
I wonder if it's better to add this under a `Note: Not recommended for Production`